financial-connections/src/main/java/com/stripe/android/financialconnections/core/FinancialConnectionsViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package com.stripe.android.financialconnections.core
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.stripe.android.financialconnections.core.Result.Fail
+import com.stripe.android.financialconnections.core.Result.Loading
+import com.stripe.android.financialconnections.core.Result.Success
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+internal abstract class FinancialConnectionsViewModel<S>(
+    initialState: S
+) : ViewModel() {
+
+    val stateFlow: MutableStateFlow<S> = MutableStateFlow(initialState)
```

Comment:
Should keep this private and expose a non-mutable `StateFlow`.

Reply:
https://github.com/stripe/stripe-android/pull/8125/commits/0f39b41568b34605bc8c295238425a6d3011e72a